vaadin-icons-flow-parent/vaadin-icons-flow/src/main/java/com/vaadin/flow/component/icon/Icon.java

```diff
@@ -31,14 +31,13 @@
 public class Icon extends AbstractIcon<Icon> {
 
     private static final String ICON_ATTRIBUTE_NAME = "icon";
-    private static final String ICON_COLLECTION_NAME = "vaadin";
+    private static final String VAADIN_ICON_COLLECTION_NAME = "vaadin";
     private static final String STYLE_FILL = "fill";
 
     /**
-     * Creates an Icon component that displays a Vaadin logo.
+     * Creates an empty Icon.
      */
     public Icon() {
-        this(VaadinIcon.VAADIN_H);
```

Comment:
This is a subtle breaking change. Can we exclude it from the PR?

Reply:
We can but it is currently quite broken when you create an empty icon and do not get an empty icon. It is a bit like `new Button()` would create a button that says "Hello World".

Reply:
To me it only looks like "We must set some icon because there is no setter for an icon", and that blocker is now removed

Reply:
I've checked this with the team and it's okay to proceed, mentioning that it's a minor behavior altering change.